portal/src/main/webapp/js/src/DataProxyFactory.js

```diff
@@ -50,10 +50,7 @@ var DataProxyFactory = (function()
 			// note that gene list is not set as a servlet param, it is passed a constructor parameter
 			var servletParams = {};
 
-			servletParams.geneticProfiles = window.QuerySession.getCancerStudyIds()[0] + "_mutations";
-
-			var caseSetId = window.QuerySession.getCaseSetId();
-			var caseIdsKey = window.QuerySession.getCaseIdsKey();
+			servletParams.geneticProfiles = geneticProfiles;
```

Comment:
Something like this would help supporting default case:

`servletParams.geneticProfiles = geneticProfiles || window.QuerySession.getCancerStudyIds()[0] + "_mutations";`